### PR TITLE
Forks/saml2 v4.9.0 candidate

### PR DESCRIPTION
**Pull request by starsinmypockets**
Use latest saml versions:
https://github.com/GSA/pysaml2/tree/datagov/max/v4.9.0
https://github.com/GSA/ckanext-saml2/tree/datagov/v0.4.0

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,6 +24,8 @@ RUN apt-get -q -y update && apt-get -q -y install \
   libpq-dev \
   libxml2-dev \
   libxslt1-dev \
+  libmagic-dev \
```
Comment (adborden) [inline Dockerfile:27]: A bit surprised to see libmagic here.

File: config/environments/docker/production.ini [hunk 2]
```diff
@@ -101,7 +101,7 @@ ckan.search.solr_commit = true
 #       Add ``datapusher`` to enable DataPusher
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
-ckan.plugins = stats text_view image_view recline_view geodatagov datagov_harvest ckan_harvester geodatagov_geoportal_harvester z3950_harvester arcgis_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query resource_proxy spatial_harvest_metadata_api recline_preview datagovtheme datajson_harvest datajson archiver qa googleanalyticsbasic extlink report broken_link_report
+ckan.plugins = stats text_view image_view recline_view geodatagov datagov_harvest ckan_harvester geodatagov_geoportal_harvester z3950_harvester arcgis_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query resource_proxy spatial_harvest_metadata_api recline_preview datagovtheme datajson_harvest datajson archiver qa googleanalyticsbasic extlink report broken_link_report saml2
```
Comment (adborden) [inline config/environments/docker/production.ini:104]: We don't want to enabled saml2 in development unless there is a way to still login with username/password. We don't have a way to setup saml2 authentication for development right now.

File: requirements.txt [hunk 3]
```diff
@@ -84,10 +84,9 @@ progressbar==2.3
 
 
 # ckanext-saml2
--e git+https://github.com/GSA/pysaml2.git@max#egg=pysaml2
+-e git+https://github.com/gsa/pysaml2@datagov/max/v4.9.0#egg=pysaml2
```
Comment (adborden) [inline requirements.txt:87]: We should probably just rename this branch `datagov/v4.9.0`.

**Comment by starsinmypockets** [comment]
@adborden this works at sandbox

**Comment by adborden** [comment]
@starsinmypockets part of https://github.com/GSA/datagov-ckan-multi/issues/346 ?

**Comment by starsinmypockets** [comment]
Closing -- duplicate of #154

**Pull request closed without merging**